src/createBot.test.ts

```diff
@@ -191,17 +193,24 @@ describe('send functions', () => {
   });
 });
 
+function log(line: string) {
+  fs.appendFileSync('./log.txt', `${line}\n`);
```

Comment:
Ah good idea, I really did not think of this since I'm only using it for my own sake. I was not familiar with how tests works and had to do some manual logging...
Maybe we can push it to a separate PR for this issue.

Reply:
Sure thing. Perhaps create an issue for tracking?